### PR TITLE
pre-release docs for publishing to Posit Cloud

### PR DESCRIPTION
**Pull request by mslynch**
Once 1.4 is released, this should move to `docs/publishing/posit-cloud.qmd`.

### CODE REVIEW

File: docs/publishing/posit-cloud.qmd
Comment (mslynch) [inline docs/publishing/posit-cloud.qmd:1]: This is based heavily on `docs/publishing/rstudio-connect.qmd`.

File: docs/publishing/posit-cloud.qmd [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+
+1.  If you don't already have one, create a token and token secret in Posit Cloud (under Account > Tokens).
+
+2.  Add the token secret to your repository's action **Secrets** (accessible within repository **Settings**). You will see a **New repository secret** button at the top right:
+
+    ![](images/gh-new-repository-secret.png){.border}
+
+    Click the button and add the API Key from step 1 as a secret named `POSIT_CLOUD_SECRET`:
+
+    ![](images/gh-action-posit-cloud-secret.png){.border}
```
Comment (cwickham) [inline docs/publishing/posit-cloud.qmd:184]: Please add alt text

**Review by cwickham — CHANGES_REQUESTED** [review]
This looks great, and I'd like to get it up into our 1.4 highlights page now. 
I've made a few specific comments on `docs/publishing/cloud.qmd`. I'll prep it's move into the highlights section, so we can get it merged once you address them.

**Pull request merged**